otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/semisl/model.py

```diff
@@ -17,6 +17,44 @@
 # pylint: disable=invalid-name
 
 _base_ = [
-    "../../base/models/mean_teacher.py",
```

Comment:
I suggest to remove this file if all model config is specified here!

Reply:
Thank you. Deleted in this [commit](https://github.com/openvinotoolkit/training_extensions/pull/1875/commits/13fa9694d5b23702bf064005474f533fab16b559).